src/sage/symbolic/expression.pyx

```diff
@@ -7656,7 +7659,7 @@ cdef class Expression(Expression_abc):
             sage: f = x^2 -2/3/x + 1
             sage: f.laurent_polynomial(QQ)
             -2/3*x^-1 + 1 + x^2
-            sage: f.laurent_polynomial(GF(19))
+            sage: f.laurent_polynomial(GF(19))                                          # needs sage.rings.finite_rings
             12*x^-1 + 1 + x^2
         """
```

Comment:
Is `GF(19)` small?

Reply:
Yes, it's small, so adding `sage.rings.finite_rings` is not necessary. I'll fix it